tests/data/taca_test_cfg.yaml

```diff
@@ -0,0 +1,89 @@
+log:
+    log_file: data/taca.log
+
+storage:
+    data_dirs:
+        - /srv/illumina/hiseq_data
+        - /srv/illumina/miseq_data
+    archive_dirs:
+        - /srv/illumina/hiseq_data/nosync
+        - /srv/illumina/miseq_data/nosync
+    irods:
+        irodsHome: /ssUppnexZone/proj/a2010002
+
+cleanup:
```

Comment:
no problem, i'll clean up the config a bit. as you've noted, this config is not actually for the test suite but rather for facilitating running an integration test.